mslice/presenters/cut_plotter_presenter.py

```diff
@@ -115,9 +135,13 @@ def add_overplot_line(self, workspace_name, key, recoil, cif=None):
             q_axis = cache.cut_axis
         x, y = compute_powder_line(workspace_name, q_axis, key, cif_file=cif)
         try:
-            y = np.array(y) * (scale_fac / np.nanmax(y))
+            if not y_has_logarithmic:
+                y = np.array(y) * scale_fac / np.nanmax(y) + datum
+            else:
+                y = self._get_log_bragg_y_coords(len(y), 0.15, datum)
```

Comment:
Can the `0.15` be moved out of this function and become a 'named variable' at the top of the file. For instance something like `BRAGG_SIZE_ON_AXES = 0.15`?

Reply:
Yep, good suggestion